### DISCUSSION TITLE
와 RKTM 업뎃함

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#338] [일반]
https://steamcommunity.com/sharedfiles/filedetails/changelog/1848471817

**Comment by RimWorldKorea** [comment]
진짜네

**Reply by zilrty** [reply]
rktm 폭풍 업데이트 중인데 rmk에 번역 존재하는거 번역 추가한거도 있고 번역 갱신에 최적화 중인데

rktm 모더랑 컨택할 생각은 없음?

**Reply by RimWorldKorea** [reply]
확인하고 협의할 내용이 한 두 가지가 될 것 같진 않아서 아직 미뤄두는 중
